pytest_localserver/http.py

```diff
@@ -11,7 +10,8 @@
 
 from werkzeug.datastructures import Headers
 from werkzeug.serving import make_server
-from werkzeug.wrappers import Response, Request
+from werkzeug.wrappers import Request
+from werkzeug.wrappers import Response
```

Comment:
TBH I'm not exactly in favor of splitting apart each import into its own separate statement like this, but it's a super minor thing so I don't mind keeping it like this. I just wanted to make a note of it so it's "on the record".

Reply:
It's because of `reorder_python_imports` pre-commit hook. `pytest` also uses this hook.
We can replace it with `isort` if you want